packages/@aws-cdk/aws-cloudwatch/lib/metric.ts

```diff
@@ -823,6 +887,70 @@ export interface CreateAlarmOptions {
   readonly datapointsToAlarm?: number;
 }
 
+/**
+ * Properties needed to make an alarm from a metric
+ */
+export interface CreateAnomalyDetectionAlarmOptions {
+
+  /**
+   * Name of the alarm
+   *
+   * @default Automatically generated name
+   */
+  readonly alarmName?: string;
+
+  /**
+   * Description for the alarm
+   *
+   * @default No description
+   */
+  readonly alarmDescription?: string;
+
+  /**
+   * Comparison to use to check if metric is breaching
+   *
+   * @default GreaterThanOrEqualToThreshold
```

Comment:
GreaterThanOrEqualToThreshold does not exist on AnomalyDetectionComparisonOperator

Reply:
Fixed, set to 

```
* @default - AnomalyDetectionComparisonOperator.LESS_THAN_LOWER_OR_GREATER_THAN_UPPER_THRESHOLD
```